ribbon/ribbon/contract.py

```diff
@@ -60,7 +60,7 @@ def __init__(self, config: ContractConfig):
                 + f"({chain.value})"
             )
 
-        if chain == Chains.FUJI:
+        if chain == Chains.FUJI or chain == Chains.BSC or chain == Chains.BSC_TESTNET:
```

Comment:
We do not want to convert the ORs into `chain in [Chains.FUJI, Chains.BSC, Chains.BSC_TESTNET]`?

Reply:
Done